cfpb/sbl-frontend
# Update the styles for an AssociatedInstitution

Closes #191 
Closes #165 

## Changes

- Refactors the AssociatedInstitution component to: 
  - Encapsulate styles
  - Adjust font-weight of the `status` field per request from the Design team
-  Plugs the updated component into the AuthenticatedLanding and UserProfile pages

## How to test this PR

1. Log in to the Platform
2. Verify the `Approved` text is no longer bold
3. Click on the `User Profile`
2. Verify the `Approved` text is no longer bold

## Screenshots
### Authenticated landing
<img width="715" alt="Screenshot 2024-01-24 at 10 14 21 AM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/a9540602-524b-438f-a8a9-d6f8f7734b4c">

### User profile
<img width="725" alt="Screenshot 2024-01-24 at 10 14 33 AM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/ff8bd8f7-56fd-4777-9612-0d7503ab4035">
